Dockerfile

```diff
@@ -0,0 +1,13 @@
+# image - sudosk/mosotf:latest
+
+# continuumio/miniconda3:4.8.2
+FROM continuumio/miniconda3@sha256:456e3196bf3ffb13fee7c9216db4b18b5e6f4d37090b31df3e0309926e98cfe2
+
+LABEL authors="sangramsahu15@gmail.com" \
+      description="Docker image containing dependencies for krassowski/multi-omics-state-of-the-field"
+
+COPY environment.yml /
+
+RUN apt-get update && \
```

Comment:
I guess will also need to install deps from [apt.txt](https://github.com/krassowski/multi-omics-state-of-the-field/blob/master/apt.txt).

I manually created a conda environment for Binder because it is faster than compiling the R dependencies, but if possible a better alternative might be to install from requirements and restore R dependencies with renv as described in our README.